src/Umbraco.PublishedCache.HybridCache/Persistence/DatabaseCacheRepository.cs

```diff
@@ -137,9 +145,9 @@ private void TruncateContent()
     public async Task<ContentCacheNode?> GetContentSourceAsync(Guid key, bool preview = false)
     {
         Sql<ISqlContext>? sql = SqlContentSourcesSelect()
-            .Append(SqlObjectTypeNotTrashed(SqlContext, Constants.ObjectTypes.Document))
-            .Append(SqlWhereNodeKey(SqlContext, key))
-            .Append(SqlOrderByLevelIdSortOrder(SqlContext));
+        .Append(SqlObjectTypeNotTrashed(SqlContext, Constants.ObjectTypes.Document))
+        .Append(SqlWhereNodeKey(SqlContext, key))
+        .Append(SqlOrderByLevelIdSortOrder(SqlContext));
```

Comment:
```suggestion
            .Append(SqlObjectTypeNotTrashed(SqlContext, Constants.ObjectTypes.Document))
            .Append(SqlWhereNodeKey(SqlContext, key))
            .Append(SqlOrderByLevelIdSortOrder(SqlContext));
```

Would prefer to keep the indentation here.